### ISSUE TITLE
convertmc image - plot geometry

### ISSUE THREAD

**Issue by nbassler** [DataMedSci/pymchelper#764]
something like

convertmc image --many "foo*.bdo" --geometry="geo.dat"

then (at least the bodies if not zones) could be outlined as well in the plot

**Comment by grzanka** [comment]
Do you mean that pymchelper should parse the geometry and then print it on top of the scorers ?

**Comment by nbassler** [comment]
Yes, something like what flair (fluka) also does for the results. Could be a student project or so.

**Comment by grzanka** [comment]
Funny enough we have borrowed part of Flair code in pymchelper as well, useful to digest binary Fluka files:
https://github.com/DataMedSci/pymchelper/tree/master/pymchelper/flair

We could then extend such feature to draw geometry on Fluka outputs as well
